py_zipkin/_encoding_helpers.py

```diff
@@ -1,12 +1,21 @@
+import json
 import socket
 from collections import namedtuple
+from enum import Enum
+
+from py_zipkin import thrift
 
 Endpoint = namedtuple(
     'Endpoint',
     ['service_name', 'ipv4', 'ipv6', 'port'],
 )
 
 
+class Encoding(Enum):
```

Comment:
add _V1 suffix here? allows you to give JSON_V2 later.. or if this is more a about json itsefl leave this alone but add a "format" thing later...

Reply:
I was thinking of keeping the version and the format separated and so add another `Version(Enum)`. But maybe it's better as you say and have them in a single struct, so we don't have to worry about users setting `V1` with `THRIFT` encoding and similar